_pages/home/spartacus-faq.md

```diff
@@ -3,131 +3,140 @@ title: Spartacus FAQ
+- On Android-based devices, Spartacus is tested with Chrome, which is based on Chromium and the Blink layout engine. Any Android browser that uses the same engine will likely work with Spartacus. Other browsers using different browser engines are not tested, but those browsers that use Webkit should work as well.
+
+### How does Spartacus compare to the SAP Commerce Cloud Accelerators?
 
 Accelerators templates (link) were introduced in SAP Commerce v4.4 to provide a ready-to-use starter implementation; they allowed partners to quickly develop an omni-channel experience by changing an example storefront. While extendable, the templates were not easily upgradable, they were JSP-based, and the storefronts were coupled tightly with the platform. Spartacus is a set of libraries that help you create a decoupled, modern, JavaScript-based storefront that is similarly extendable but vastly more upgradable. Spartacus is the strategic way forward for maintaining a customizable storefront with SAP Commerce Cloud. Please see the article “Choosing Which Storefront to Use for Your SAP Commerce Cloud Solution” for more information on this topic. 
 
-### Are there features that are missing in Spartacus that exist in the SAP Commerce Cloud Accelerators? 
+### Are there features that are missing in Spartacus that exist in the SAP Commerce Cloud Accelerators?
 
 At the 1.0 release, Spartacus will not have feature parity when compared with all the Accelerators available in SAP Commerce Cloud. The focus to start is to achieve feature parity with the B2C accelerator. Feature parity with B2B, China and Industry accelerators will come in the future. Feature parity will be documented. 
 
-### Can I run Spartacus and an Accelerator storefront at the same time? 
+### Can I run Spartacus and an Accelerator storefront at the same time?
 
 Yes. Spartacus-based storefronts are decoupled from SAP Commerce Cloud, so you can run both your Accelerator template-based store and Spartacus storefront. In SAP Commerce Cloud, this would be a matter of configuring your code, aspects, and endpoints configured correctly. See the product documentation here and here for more. 
 
-### Does this mean the Accelerators will be deprecated? 
+### Does this mean the Accelerators will be deprecated?
 
 Eventually, but that is many years out. Our goal is to move all customers to Spartacus, or at least JavaScript-based storefronts. All current development efforts are focused on improving Spartacus. For the moment, Accelerator templates are receiving critical and blocker bug or security fixes only. At the very least, Accelerator templates won’t be deprecated until Spartacus achieves feature parity. Spartacus is the strategic way forward for maintaining a customizable storefront with SAP Commerce Cloud. Please see the article “Choosing Which Storefront to Use for Your SAP Commerce Cloud Solution” for more information on this topic. 
```

Comment:
I'd be careful with "many years", it might come sooner than we think.

Reply:
Bill's replacement text: "Eventually. We will announce deprecation and removal plans at a later date."